src/api/operator/types.ts

```diff
@@ -38,6 +40,30 @@ export type RawOrder = {
   signature: string
 }
 
+/**
+ * Enriched Order type.
+ * Applies some transformations on the raw api data.
+ * Some fields are kept as is.
+ */
+export type Order = Pick<RawOrder, 'owner' | 'uid' | 'appData' | 'kind' | 'partiallyFillable' | 'signature'> & {
+  creationDate: Date
+  expirationDate: Date
+  buyTokenAddress: string
+  sellTokenAddress: string
+  buyAmount: BigNumber
+  sellAmount: BigNumber
+  executedBuyAmount: BigNumber
+  executedSellAmount: BigNumber
+  feeAmount: BigNumber
+  executedFeeAmount: BigNumber
+  cancelled: boolean
+  // status: OrderStatus // better not have the status because it depends on `expirationDate`
```

Comment:
are you keeping these commented props for a reason

Reply:
So we can talk about it :)
Will remove